src/android/HealthPlugin.java

```diff
@@ -744,7 +745,11 @@ else if (mealt == Field.MEAL_TYPE_SNACK)
           }
           obj.put("value", bpobj);
           obj.put("unit", "mmHg");
-        }
+        }  else if (dt.equals(DataType.TYPE_SLEEP_SEGMENT)) {
+          int sleepSegmentType = datapoint.getValue(Field.FIELD_SLEEP_SEGMENT_TYPE).asInt();
```

Comment:
map the integer to the following strings:
SLEEP  -> sleep
AWAKE  -> sleep.awake
OUT_OF_BED -> sleep.outOfBed
SLEEP_DEEP -> sleep.deep
SLEEP_LIGHT -> sleep.light
SLEEP_REM -> sleep.rem

Reply:
Hi! 

Thank you for your response.

I just added a commit with your proposed change.